packages/nextjs/components/scaffold-stark/Input/AddressInput.tsx

```diff
@@ -21,7 +21,25 @@ export const AddressInput = ({
 
   const handleChange = useCallback(
     (newValue: Address) => {
-      //setEnteredEnsName(undefined);
+      const sanitizedValue = newValue.toLowerCase();
+
+      if (sanitizedValue === "0x") {
+        onChange("0x0" as Address);
+        return;
+      }
+
+      if (/^0x.*0x/.test(sanitizedValue.slice(2))) {
+        return;
+      }
```

Comment:
this still does not cover the cases. would be best to just use a regex maybe? e.g. invalid stuff like `0x1235x21412xyz` might happen.

Reply:
Ok i working in this